src/Prismic/Dom/RichText.php

```diff
@@ -277,9 +277,16 @@ private static function serialize($element, $content, $linkResolver, $htmlSerial
             case 'o-list-item':
                 return nl2br('<li' . $classCode . '>' . $content . '</li>');
             case 'image':
+                $img = '<img src="' . $element->url . '" alt="' . htmlentities($element->alt) . '">';
+
+                $link = property_exists($element, 'linkTo') ? Link::asUrl($element->linkTo, $linkResolver) : null;
+
+                $target     = property_exists($element, 'linkTo') ? ($element->linkTo->target ?? null) : null;
+                $targetCode = $target ? ' target="' . $target . '"' : '';
```

Comment:
@gsteel Should I wrap this target in a `htmlentities()` 🤔

Reply:
Anything destined for output should be escaped using a tool that considers context. I advocate the use of [Laminas Escaper](https://docs.laminas.dev/laminas-escaper/intro/) - In this case, I would build an array of attributes and use `Escaper::escapeHtmlAttr()` on each. The client I maintain uses the escaper component [here](https://github.com/netglue/prismic-client/blob/1.6.x/src/Serializer/HtmlSerializer.php).

Reply:
Escaped in #198